src/screens/IndexScreen.js

```diff
@@ -145,6 +148,14 @@ export const IndexScreen = ({ navigation, route }) => {
       [QUERY_TYPES.EVENT_RECORDS]: showEventsFilter,
       [QUERY_TYPES.NEWS_ITEMS]: showNewsFilter
     }[query];
+  const htmlContentName =
+    (query === QUERY_TYPES.POINTS_OF_INTEREST && poiListIntro?.[queryVariables.category]) || '';
```

Comment:
we do not need a fallback here. if we do not have the POI query and a `poiListIntro`, `htmlContentName` should be undefined

Reply:
I fixed it.
https://github.com/smart-village-solutions/smart-village-app-app/commit/a30ae59e770597a7b078e8ba7e44919e29c990c6